addons/mail/static/src/discuss/google_translate/common/message_patch.js

```diff
@@ -0,0 +1,27 @@
+/* @odoo-module */
+
+import { Message } from "@mail/core/common/message";
+import { patch } from "@web/core/utils/patch";
+import { markup } from "@odoo/owl";
+
+patch(Message.prototype, {
```

Comment:
As it's not related to channels in any way, I would not have this as patch in discuss, but everything in mail itself.

Reply:
Everything has been merged to the existing files of mail/static/src/core/common